src/config/loadGitHubAppsFromEnvironment.ts

```diff
@@ -1,36 +1,72 @@
-import { AppAuthStrategyOptions } from '@/types';
+import {
+  AppAuthStrategyOptions,
+  GitHubIssuesSomeoneElseCaresAbout,
+} from '@/types';
 
 export class GitHubApp {
+  num: number;
   org: string;
   auth: AppAuthStrategyOptions;
-  project;
+  project: GitHubIssuesSomeoneElseCaresAbout;
 
-  constructor(org: string, opts) {
-    this.org = org;
-    this.auth = {
-      appId: Number(opts.GH_APP_IDENTIFIER),
-      privateKey: opts.GH_APP_SECRET_KEY.replace(/\\n/g, '\n'),
-    };
-    this.project = {
-      node_id: opts.ISSUES_PROJECT_NODE_ID || 'PVT_kwDOABVQ184AOGW8',
-      product_area_field_id:
-        opts.PRODUCT_AREA_FIELD_ID || 'PVTSSF_lADOABVQ184AOGW8zgJEBno',
-      status_field_id: opts.STATUS_FIELD_ID || 'PVTSSF_lADOABVQ184AOGW8zgI_7g0',
-      response_due_date_field_id:
-        opts.RESPONSE_DUE_DATE_FIELD_ID || 'PVTF_lADOABVQ184AOGW8zgLLxGg',
-    };
+  constructor(obj) {
+    this.num = obj.num;
+    this.org = obj.org;
+    this.auth = obj.auth;
+    this.project = obj.project;
   }
 }
 
-export class GitHubAppsRegistry {
-  apps: Map<string, GitHubApp>;
+class GitHubAppsConfigHelper {
```

Comment:
I split out a separate class for the config pass, it felt too messy lumped in with the app registry.